tox.ini

```diff
@@ -1,20 +1,21 @@
 [tox]
 envlist =
-    py{37}-codestyle,
-    py{37}-django{22,32}
+    py{38}-codestyle,
+    py{38,39,310}-django{32,42}
 
 [testenv]
 setenv =
     PYTHONDONTWRITEBYTECODE=1
 deps =
-    django18: Django>=1.8,<1.9
-    django19: Django>=1.9,<1.10
-    django22: Django>=2.2,<3.0
     django32: Django>=3.2,<4.0
+    django42: Django>=4.2,<5.0
     -rrequirements.txt
-commands = ./runtests.py {posargs}
+commands = python runtests.py {posargs}
 
 
-[testenv:py37-codestyle]
+[testenv:py38-codestyle]
 deps = -rrequirements.txt
-commands = multilint
+commands =
+    black -S gargoyle tests runtests.py setup.py
+    isort gargoyle tests runtests.py setup.py
+    flake8
```

Comment:
Whats the idea behind removing `multilint` here?

Reply:
As far as I know, `multilint` is a legacy project that runs up until python 3.8, but its main usecase was linting python 2 and 3 projects.
I tried to replicate the same behavior using these three tools, it might be useful if/when we bump to a newer python version (3.8 EOL is November 2024)